README nits (#2)
stytchauth/stytch-netlify-example: README.md

```

## Running the app on Netlify
Click the button below to deploy this application to Netlify. This will create a new Netlify site and deploy the application to it. You will be prompted to enter your Stytch [API Keys](https://stytch.com/dashboard/api-keys). 

> Note, you'll need to add your Netlify's site domain to the [Redirect URLs](https://stytch.com/dashboard/redirect-urls) and [Authorized applications](https://stytch.com/dashboard/sdk-configuration) in your Stytch Dashboard that you previously added `http://localhost:8888` to. 

Reviewer: ```suggestion
Note, you'll need to add your Netlify's site domain to the [Redirect URLs](https://stytch.com/dashboard/redirect-urls), e.g. `https://main--wonderful-salmiakki-f95458.netlify.app/authenticate` and [Authorized applications](https://stytch.com/dashboard/sdk-configuration) in your Stytch Dashboard, e.g. `https://main--wonderful-salmiakki-f95458.netlify.app`. 
```
